page-faq.php

```diff
@@ -278,25 +310,31 @@
 </ul>
 
 <a name="closedapps"></a>
-<h3>Can I write closed source or proprietary apps for ownCloud?</h3>
+<h3>Can I write closed source or proprietary apps for ownCloud Server?</h3>
```

Comment:
I rather not have people read that item at all if they want to develop an external app, but just go right away to the one talking about 3rd party apps. So I'll keep it in ;-)